lib/components/term.js

```diff
@@ -61,6 +65,60 @@ const getTermOptions = props => {
   };
 };
 
+const getFileData = soundFileLocation => {
+  return new Promise(async resolve => {
+    const isRemoteUrl = isValidUrl(soundFileLocation);
```

Comment:
One problem I see with accepting remote URLs is that your terminal won't have any sound if you start without internet access?

We could cache it and add more logic, but I think it's best just to only accept local files and let the user download it manually? This is a very novel feature that not many will use but it can cause a lot of issues.

Reply:
Thats a good point about using remote urls and lack of internet at the time of spawning the terminal. Maybe for "restoring functionality" we can stick with only local files.

Reply:
yes, also it would slow down startup significantly to download the sound every launch 👍  I don't think hyper should support remote URLs for this

Reply:
Loading remote sound can be done async. So it would not slow down startup but there is a lot of case to deal with: during loading, after a failure etc...

I agree it could be better to stick with local files only.